content/docs/faq/README.md

```diff
@@ -18,6 +18,37 @@ face:
 
 This is a feature in cert-manager starting in `v0.16` using the `cmctl` CLI. More information can be found on [the renew command's page](../usage/cmctl.md#renew)
 
+### When do certs get re-issued?
+
+To determine if a certificate needs to be re-issued, cert-manager looks at the the spec of `Certificate` resource and latest `CertificateRequest`s as well as the data in `Secret` containing the X.509 certificate.
+
+Issuance process will always get triggered if:
+- the `Secret` named on `Certificate`'s spec, does not exist, is missing private key or certificate data or contains corrupt data
+- private key stored in the `Secret` does not match private key spec on `Certificate`
+- public key of the issued certificate does not match the private key stored in the `Secret`
+- cert-manager issuer annotations on the `Secret` do not match the issuer specified on `Certificate`
+- DNS names, IP addresses, URLS or email addresses on the issued certificate do not match those on the `Certificate` spec
+- certificate needs to be renewed
```

Comment:
nitpick (non-blocking): I think with a slight reword this would read a little more naturally, but it's totally fine as is!

suggestion (non-blocking): Also suggesting a little extra! We mention `cmctl renew` below but I wonder if it might be worth also mentioning it here since I feel like people are more likely to read bullet points than prose!

```suggestion
The issuance process will always get triggered if the:

- `Secret` named on `Certificate`'s spec, does not exist, is missing private key or certificate data or contains corrupt data
- private key stored in the `Secret` does not match the private key spec on `Certificate`
- public key of the issued certificate does not match the private key stored in the `Secret`
- cert-manager issuer annotations on the `Secret` do not match the issuer specified on the `Certificate`
- DNS names, IP addresses, URLS or email addresses on the issued certificate do not match those on the `Certificate` spec
- certificate needs to be renewed (because it's going to expire)
- certificate has been marked for renewal manually [using `cmctl`](../usage/cmctl.md#renew)
```